src/internal/components/InternalEditor.tsx

```diff
@@ -97,41 +96,62 @@ export const InternalEditor = ({
   };
 
   const handleSave = async (data: Data) => {
+    if (themeModeActive) {
+      devLogger.logFunc("saveStyles");
+      // save styles
+      return;
+    }
     devLogger.logFunc("saveVisualConfigData");
     saveVisualConfigData({
       payload: { visualConfigurationData: JSON.stringify(data) },
     });
   };
 
-  const change = async () => {
```

Comment:
I think I had added this because there were issues without it. If thinks are working fine then great, just be on the lookout.

Reply:
I'll do this in another PR and check that it's ok